README.md

```diff
@@ -415,19 +415,24 @@ We strive to completely cover all applicable issues with these core labels:
    * __[S-Tolerable](https://github.com/vector-im/element-web/labels/S-Tolerable):__
      Purely cosmetic or low / no impact to users
 
-3. __Priority__ — All issues which are not `T-Other` are assigned a priority:
-   * __[P1](https://github.com/vector-im/element-web/labels/P1):__ Next
-   * __[P2](https://github.com/vector-im/element-web/labels/P2):__ Later
-   * __[P3](https://github.com/vector-im/element-web/labels/P3):__ Eventually
-   * __[P4](https://github.com/vector-im/element-web/labels/P4):__ Interesting —
-     Not yet scheduled, will accept patches
-   * __[P5](https://github.com/vector-im/element-web/labels/P5):__ Dubious —
-     Will not schedule, would consider patches
+3. __Prevalence__ — All issues labeled `T-Defect` are also assigned a prevalence:
+   * __[P-High](https://github.com/vector-im/element-web/labels/P-High):__ Affects most users regularly or impacts most users' first experience (e.g. registration) 
+   * __[P-Medium](https://github.com/vector-im/element-web/labels/P-Medium):__ Affects some users regularly or most users rarely
+   * __[P-Low](https://github.com/vector-im/element-web/labels/P-Low):__ Most users are unlikely to come across this or it is a corner case
 
 4. __Area__ — Most issues are assigned one or several "areas" using one of the
    many `A-` prefixed labels, e.g. `A-Composer` or `A-Spaces`. Each area label
    maps to a group of features or portion of the UI surface in the app.
 
+The Priority label is now deprecated. Defects will now be classed based on their severity and frequency
+| Labels    | Equivalent priority | What it means |
```

Comment:
I am quite confused... sorry, could we get a section explaining exactly when is Frequency used and how does it compare to Prevalence

(sorry, maybe this is obvious to native English speakers)

Reply:
I can improve the language here